### ISSUE TITLE
Make Loss functions customizable

### ISSUE THREAD

**Issue by akanimax** [akanimax/open-styleganv2-pytorch#4]
Loss functions are not customizable here :disappointed:! 
Perhaps a feature for the next codathon?

_Originally posted by @akanimax in https://github.com/akanimax/open-styleganv2-pytorch/pull/1#discussion_r479823293_
